alf/trainers/policy_trainer.py

```diff
@@ -973,6 +973,12 @@ def _step(algorithm,
         time.sleep(sleep_time_per_step)
 
     next_time_step = env.step(policy_step.output)
+    experience = Experience(
```

Comment:
you can use `make_experience`: `make_experience(time_step, policy_step, policy_state)`
Note that time_step should be used instead of next_time_step

Reply:
Updated accordingly.